projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts

```diff
@@ -303,7 +303,7 @@ export class TuiInputCardGroupedComponent
         return this.cardPrefilled ? `*${this.card.slice(-4)}` : '*';
     }
 
-    @HostListener('keydown.esc')
+    @HostListener('document:keydown.escape')
```

Comment:
This would cause change detection on every escape keypress on the document. While this component is rarely used, I'd still like to avoid that.

Reply:
fixed